.github/workflows/ci.yml

```diff
@@ -104,7 +104,8 @@ jobs:
     needs:
       - cargo-toml-fmt-check
       - set-env-vars
-    if: needs.set-env-vars.outputs.IS_RELEASE != 'true'
+      - check-is-semver-branch
+    if: ${{ (needs.set-env-vars.outputs.IS_RELEASE_OR_MASTER != 'true') && (needs.check-is-semver-branch.outputs.is_semver_branch != 'true' ) }}
```

Comment:
@deekerno Looks fine. Is there any way to make this a single check instead of an `&&` check? Would help with readability. Some type of `IS_RELEASE_OR_MASTER_OR_SEMVER_BRANCH` ?

Reply:
@ra0x3 I tried a few different ways, but it looks like we can't do the regex check in the `env` step to create a `IS_RELEASE_OR_MASTER_OR_SEMVER_BRANCH` env var. 👎🏽